react-app/src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+/*
+This is the landing page that will display when users first open the app. It has a title and a menu.
+*/
+
+import React from "react";
+import { LinkContainer } from 'react-router-bootstrap'
+// import { Button } from 'react-bootstrap'
+
+import '../stylesheets/Home.css';
+
+class Home extends React.Component {
+    render() {
+        this.whatever = 10;
```

Comment:
Same comment about removing this unused variable

Reply:
Resolved by changing the Home page from a class to a function, since it doesn't need to use this or state.